platform/muen/componolit_runtime.adb

```diff
@@ -0,0 +1,96 @@
+
+with System;
+with SK.CPU;
+with Debuglog.Client;
+with Gnat_Helpers;
+
+package body Componolit_Runtime with
+   SPARK_Mode
+is
+
+   procedure Log_Debug (S : String)
+   is
+   begin
+      Debuglog.Client.Put_Line
+         ("Info: " & S);
+   end Log_Debug;
+
+   procedure Log_Warning (S : String)
+   is
+   begin
+      Debuglog.Client.Put_Line
+         ("Warning: " & S);
+   end Log_Warning;
+
+   procedure Log_Error (S : String)
+   is
+   begin
+      Debuglog.Client.Put_Line
+         ("Error: " & S);
+   end Log_Error;
+
+   procedure Unhandled_Terminate with
+      Export,
+      Convention => C,
+      External_Name => "__gnat_unhandled_terminate";
+
+   procedure Unhandled_Terminate
+   is
+   begin
+      SK.CPU.Stop;
+   end Unhandled_Terminate;
+
+   function Personality (Version : Integer;
+                         Phase   : Long_Integer;
+                         Class   : Natural;
+                         Exc     : System.Address;
+                         Context : System.Address) return Gnat_Helpers.URC with
+      Export,
+      Convention => C,
+      External_Name => "__gnat_personality_v0";
+
+   function Personality (Version : Integer;
+                         Phase   : Long_Integer;
+                         Class   : Natural;
+                         Exc     : System.Address;
+                         Context : System.Address) return Gnat_Helpers.URC
+   is
+   begin
+      SK.CPU.Stop;
```

Comment:
This function is a function to match with the GNAT ABI. In theory it should never be called since we never throw an exception but in case it is ever calles we want it to kill the subject since there is no other exception handler on Muen.